hnn_core/parallel.py

```diff
@@ -7,20 +7,85 @@
 
 from neuron import h
 
-rank = 0
-nhosts = 1
-pc = h.ParallelContext(nhosts)
-pc.done()
-rank = int(pc.id())
-cvode = h.CVode()
+pc = None
+last_network = None
+rank = None
 
 
-def create_parallel_context(n_jobs=1):
-    """Create parallel context."""
-    rank = int(pc.id())     # rank or node number (0 will be the master)
+def shutdown():
+    """Tell nrniv processes to terminate. Calling script will also terminate"""
+    pc.done()
+    h.quit()
 
-    if rank == 0:
-        pc.gid_clear()
+
+def get_nhosts():
```

Comment:
private function ...

Reply:
This is useful in combination with get_rank(), so either both should be exposed (this one added to __init__.py) or none.